Bunkum.Core/Endpoints/Middlewares/MainMiddleware.cs

```diff
@@ -292,7 +292,7 @@ private bool TryAddBodyToInvocation(EndpointAttribute attribute, Type paramType,
             }
             catch (Exception e)
             {
-                this._logger.LogError(BunkumCategory.UserContent, $"Failed to parse object data: {e}\n\nXML: {body}");
+                this._logger.LogError(BunkumCategory.UserContent, $"Failed to parse object data: {e}\n\nXML: {Encoding.UTF8.GetString(body.ToArray())}");
```

Comment:
This won't always be a MemoryStream in the future (see #67), so I'd prefer if we read everything out once at the top of the function and then used that for everything.

Might as well refactor this function while we're here